roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1090,7 +1090,7 @@ def set_container_facts_if_unset(facts):
         if 'ovs_image' not in facts['node']:
             facts['node']['ovs_image'] = ovs_image
 
-    if facts['common']['is_containerized']:
+    if facts['common']['is_containerized'] in ['true', 'True']:
```

Comment:
it would probably be better to use strtobool when setting this value, similar to: https://github.com/abutcher/openshift-ansible/blob/bz1313169-oadm/roles/openshift_facts/library/openshift_facts.py#L769